pkgs/by-name/sp/spacectl/package.nix

```
@@ -17,6 +17,13 @@ buildGoModule rec {
 
   vendorHash = "sha256-SYfXG6YM0Q2rCnoTM2tYvE17uBCD8yQiW/5DTCxMPWo=";
 
+  postInstall = ''
```

We're running the built binary to generate these completions, so we need to make sure the system we're building on can execute code meant for the system it's building for (i.e., when cross compiling for aarch64-linux, can x86_64-linux run that code?)

We can do this with the following conditional
```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

---

😮